lib/promise-any.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+module.exports = function (arrayOfPromises) {
+  let promiseAny = new Promise((resolveAny, rejectAny) => {
+    let remainingPromises = arrayOfPromises.length;
+    let resolved = false;
+    let rejectList = [];
+
+    for (let promise of arrayOfPromises) {
+      promise
```

Comment:
Check if it's a promise first. If it's not, return a resolved promise with the value.